tools/profiling/memory/memory_diff.py

```diff
@@ -165,8 +165,12 @@ def _run():
         text += "{}: {}\n".format(key, value)
 else:
     print(cur, old)
-    call_diff_size = 0
-    channel_diff_size = 0
+    call_decrease = 0
+    call_increase = 0
+    channel_decrease = 0
+    channel_increase = 0
+    call_significant = 64
```

Comment:
Any idea what the variance is across benchmarks? I haven't seen the raw data, and I'm wondering if this threshold will be above the noise. This seems like a fine goal, I'm just curious if we'll still see a lot of label churn.

Reply:
Upped it to 100... we're seeing about 65 bytes variance right now afaict